gui-wm/wayfire/wayfire-0.7.3.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 2019-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson toolchain-funcs
+
+DESCRIPTION="compiz like 3D wayland compositor"
+HOMEPAGE="https://github.com/WayfireWM/wayfire"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/WayfireWM/${PN}.git"
+else
+	SRC_URI="https://github.com/WayfireWM/${PN}/releases/download/v${PV}/${P}.tar.xz"
+	KEYWORDS="amd64 ~arm64 ~riscv ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="debug +gles +system-wfconfig +system-wlroots X"
+
+DEPEND="
+	dev-libs/libevdev
+	dev-libs/libinput
+	dev-libs/wayland
+	gui-libs/gtk-layer-shell
+	media-libs/glm
+	media-libs/mesa:=[gles2,wayland,X?]
+	media-libs/libglvnd[X?]
+	media-libs/libjpeg-turbo
+	media-libs/libpng
+	media-libs/freetype:=[X?]
+	x11-libs/libdrm
+	x11-libs/gtk+:3=[wayland,X?]
+	x11-libs/cairo:=[X?,svg]
+	x11-libs/libxkbcommon:=[X?]
+	x11-libs/pixman
+	X? (
+		x11-base/xwayland
+		x11-libs/libxcb
+	)
```

Comment:
```
meson.build:141:2: ERROR: Problem encountered: Cannot enable Xwayland in wayfire: wlroots has been built without Xwayland support
```
```
[ERROR] >>> Not all runs were successful.
atom: =gui-wm/wayfire-0.7.3, USE flags: 'X -gles -system-wfconfig system-wlroots'
atom: =gui-wm/wayfire-0.7.3, USE flags: 'X -gles system-wfconfig -system-wlroots'
atom: =gui-wm/wayfire-0.7.3, USE flags: 'X -gles system-wfconfig system-wlroots'
```
So `+X` seems to fail, apparently to a missing dependency.

Reply:
I did more USE on/off tests, after all applying suggestions and changes. +X works now.